### ISSUE TITLE
pipeline show doesn't display dependencies

### ISSUE THREAD

**Issue by eliorc** [iterative/dvc#2341]
**Setup** - Running on version 0.53.2, inside of a docker container, where container runs on Ubuntu 16.04 and the host also

I have created a file using the standard `dvc run -d ... -O ...` (notice I do opt for `-O` since the output will be versioned in git), and the resulting `.dvc` file has the following contents

```
md5: 40da64f2840947559287922675c52006
cmd: python scripts/create_dataset.py /opt/data/dataset.csv
wdir: ../../../..
deps:
- md5: 6fb9ea7998edffee57c6affc3cedfb1f
  path: /opt/data/data1.csv
- md5: 6a2dbe6e3362e871c8e019b628a3928b.dir
  path: /opt/data/data2.csv
- md5: 9968f15e5d0e52fff7dd8228322227b4.dir
  path: /opt/data/data_dir
- md5: 87cb2324a47914d995c9110c30fcaa0f
  path: /opt/scripts/create_dataset.py
outs:
- md5: 62ef5271e749f086ef0dc19aee6d279a
  path: /opt/data/dataset.csv
  cache: false
  metric: false
  persist: false
```
Also, the files specified in `-d` are not DVC versioned, they are also tracked normally in git.

The problem is, that when I try to visualize, all I see is the last step, disregarding the dependencies, so the output of `dvc pipeline show --ascii /opt/data/dataset.csv.dvc`

```
+--------------------------------------------------+
| data/dataset.csv.dvc |
+--------------------------------------------------+
```

And none of the dependencies are present - Why aren't the dependencies showing up?

(filenames and paths have been changed for privacy reasons, so if the `wdir` doesn't add up, it's ok)

**Comment by efiop** [comment]
Hi @eliorc !

Looks like `pipeline show` is not able to build the DAG properly. Could you show how dvcfile for the previous stage in the pipeline looks like please? Btw, could you tell a little more about your scenario? Why are you using abs paths? Data is in /opt/data, but where is the dvc project itself?

**Comment by eliorc** [comment]
Ok, as I said, please disregard the paths since they are changed for privacy reasons and can be assumed to be right. Yes all of the `/opt` are real, I'm mapping my application into the `/opt` path inside the container.

There was nothing before this step, this is the first dvc command I am calling (after `dvc init`) - I am not using the `dvc add` and am using the `-O` because ALL of the files I'm inputting/outputting are bit tracked by git and git lfs so I do not need the data storage feature.
So basically, all of the files in the dependencies have no `.dvc` file attached to them (since they were not created by `dvc run` and I'm not using `dvc add`).

So to sum up, the scenario was

1. Start a Git repo
2. Run DVC init
3. Download some files from the internet
4. Write a script
5. Run `dvc run` using `-d` on the files from the internet and running the script written in 4
6. Run `dvc pipeline...` which didn't show all the dependencies

**Comment by efiop** [comment]
@eliorc Right, so if there are no other stages then that output of `pipeline show` is expected, since it is showing your pipeline stages and not specific dependency files :slightly_smiling_face: . We have a ticket opened for showing particular dependencies though https://github.com/iterative/dvc/issues/1065, please feel free to like and/or comment there, so we know there is interest.

> Why are you using abs paths?? I didn't at first, since it didn't work I switched to abs paths to see if it will fix it

Hm, that is weird, do you remember what didn't work? Maybe you remember a particular error or have an error log?

**Comment by eliorc** [comment]
When I meant it didn't work, I referred to the same issue, where it doesn't show up in the dependencies... If this is the expected behavior then I'll close it and comment in the issue you referred to, thanks
